tasks/release.py

```diff
@@ -749,7 +751,39 @@ def cleanup(ctx):
     if not match:
         raise Exit(f'Unexpected version fetched from github {latest_release}', code=1)
     version = _create_version_from_match(match)
-    _update_last_stable(ctx, version)
+    current_milestone = _update_last_stable(ctx, version)
+
+    # create pull request to update last stable version
+    main_branch = "main"
+    cleanup_branch = f"release/{version}-cleanup"
+    ctx.run(f"git checkout -b {cleanup_branch}")
+    ctx.run("git add release.json")
+
+    commit_message = f"Update last_stable to {version}"
+    ok = try_git_command(ctx, f'git commit -m "{commit_message}"')
```

Comment:
NIT: (to avoid expansions)
```suggestion
    ok = try_git_command(ctx, f"git commit -m '{commit_message}'")
```